### DISCUSSION TITLE
Mdc Extractor

### DISCUSSION THREAD

**Discussion by arkaroy135** [unocss/unocss#3281] [Q&A] [answered]
Can anyone help me how to use MDC Extractor ? I followed the steps mentioned in the documentation but it is not working.

**Comment by vabatta** [comment]
What is your setup? Are you using `nuxt-content`?

**Reply by arkaroy135** [reply]
Yes. I'm using```nuxt-content```

**Reply by vabatta** [reply]
You need to add the extractor for MDC [as described in the doc](https://unocss.dev/extractors/mdc) and the following snippet to include the `.md*` files from `nuxt-content` to be processed by the UnoCSS and the extractor.

```js
content: {
  filesystem: ["./content/**/*.{md,mdc,markdown}"],
}
```

**Reply by zdv1g** [reply]
You're wrong!
Nuxt content does not have property filesystem
Docs: https://content.nuxt.com/get-started/configuration#watch

Still not working
Nuxt 3.9
Nuxt-content 3.10



Nuxt config
```
export default defineNuxtConfig({
  devtools: { enabled: true },
  modules:['@nuxt/content','@unocss/nuxt'],
})
```

Uno config:
```
export default defineConfig({
    shortcuts: {
        // shortcuts to multiple utilities
        'btn': 'py-2 px-4 font-semibold rounded-lg shadow-md',
        'btn-green': 'text-white bg-green-500 hover:bg-green-700',
        // single utility alias
        'red': 'text-red-100',
    },
    extractors: [
        extractorMdc(),
    ],
})
```

**Reply by zdv1g** [reply]
#3462

**Comment by zdv1g** [comment, marked as answer]
Answer here #3462 @arkaroy135

**Reply by arkaroy135** [reply]
Got it !!! Thank you very much. 
Wish you a Merry Christmas and a very Happy New Year !

**Reply by zdv1g** [reply]
Mark as answered
